docs/install.md

```diff
@@ -45,7 +45,7 @@ You can deploy the drivers using Helm.
 The default values mimic the config of the drivers in ACK cluster.
 We provides some configuration presets. Select one of them:
 * values-ecs.yaml: for deploy on self-built cluster on ECS.
-* values-nonecs.yaml: for deploy on non-ECS cluster. Disk driver is disabled; NAS and OSS drivers only support static volumes.
```

Comment:
Why only support static volumes?

Reply:
Because controller was not installed previously. Now that it is enabled, we should support dynamic provisioning.